libs/designer-ui/src/lib/mixedinputeditor.tsx

```diff
@@ -0,0 +1,110 @@
+import type React from 'react';
+import { useEffect, useState } from 'react';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+import type { ChangeHandler, ValueSegment } from '@microsoft/logic-apps-shared';
+import { SchemaEditor } from './schemaeditor';
+import { FloatingActionMenuInputs } from './floatingactionmenu/floatingactionmenuinputs';
+
+interface MixedInputEditorProps {
+  initialValue: ValueSegment[];
+  onChange: ChangeHandler;
+  supportedTypes?: string[];
+  isRequestApiConnectionTrigger?: boolean;
+  label?: string;
+  readonly?: boolean;
+  useStaticInputs: boolean;
+}
+
+// Function to check if `initialValue` contains "useSchemaEditor": true
+const checkIsSchemaEditor = (initialValue: ValueSegment[]): boolean => {
```

Comment:
You might want to put this into a helper file and export it, if you are required to write unit tests for this; by doing this you get get some more line coverage if you can test these. Right now they can only be tested indirectly via testing the component